src/instructlab/sdg/llmblock.py

```diff
@@ -78,7 +78,6 @@ def __init__(
             "model": self.ctx.model_id,
             "temperature": 0,
             "max_tokens": 12000,
-            "n": self.ctx.num_instructions_to_generate,
```

Comment:
quoting the commit message i made this change in ...

```
I made some past changes to how we set `n` that were not correct. The
fixes here include:

 - Re-add the one place where `n` was hard coded to 10. This was
   intentional and should be kept as-is.

 - Fix the `n` logic to be:

   - use what's specified for `n` in config if present
   - otherwise set it to 1

We never want to specify n>1 when also using a prompt that makes use
of `num_samples`, as we effectively end up with `n` * `num_samples`
results.

This restores intended behavior of the `full` pipeline, but it also
breaks applying `--num-instructions` from the CLI to be the `n` value
used with the simple pipeline. That needs to be fixed in a follow-up
commit.
```

so now I need to figure out something for the simple pipelines so that `n` is set based on runtime configuration (num_instructions_to_generate), but only for specific blocks where we want that to occur (the simple pipelines).

Reply:
follow up issue filed here so it doesn't have to block merge of this if everything else is working -- https://github.com/instructlab/sdg/issues/130